airbyte-integrations/connectors/source-mssql/src/testFixtures/java/io/airbyte/integrations/source/mssql/MsSQLTestDatabase.java

```diff
@@ -71,16 +71,27 @@ public Consumer<MSSQLServerContainer<?>> modifier() {
 
   static public MsSQLTestDatabase in(final BaseImage imageName, final ContainerModifier... modifiers) {
     final var container = new MsSQLContainerFactory().shared(imageName.reference, modifiers);
-    final var testdb = new MsSQLTestDatabase(container);
+    final MsSQLTestDatabase testdb;
+    if (false) {
```

Comment:
define a static var, with commentary explaining its purpose, instead of magic value